src/components/filters/filter_menu/filter_menu.ts

```diff
@@ -310,6 +310,7 @@ export class FilterMenu extends Component<Props, SpreadsheetChildEnv> {
     this.props.onClosed?.();
   }
 }
+FilterMenu.props = { filterPosition: Object, onClosed: { type: Function, optional: true } };
```

Comment:
Prettier ? Why is this one in one line, where `FilterIconsOverlay.props` which is shorter is on mutliple lines

Reply:
Prettier accept both, I will change this one however